scripts/build_both.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Set the output binary name
+OUTPUT_BINARY_SUPPLIER="build/supplier/bot"
+OUTPUT_BINARY_DISTRIBUTOR="build/distributor/bot"
```

Comment:
Why do we need this file? I don't get why we would need to build two identical binaries.